training/model_evaluation/environment/asset.yaml

```diff
@@ -1,5 +1,5 @@
 name: model-evaluation
-version: 1
+version: 2
 type: environment
 spec: spec.yaml
 extra_config: environment.yaml
```

Comment:
(nit) add new line

Reply:
you might want to consider auto versioning for your environment

Reply:
Hi @vizhur, we moved back to numbering because there was some issue with auto versioning which was causing env build to fail last time.

Reply:
Switched to "auto" as Sasidhar has fixed the issue